daemon/opensnitchd.service

```diff
@@ -1,15 +1,37 @@
 [Unit]
 Description=Application firewall OpenSnitch
 Documentation=https://github.com/evilsocket/opensnitch/wiki
+Documentation=man:systemd.special
+Documentation=man:systemd.service
+Documentation=man:systemd.exec
+Documentation=man:systemd.unit
+
+DefaultDependencies=no
+Before=network-pre.target shutdown.target
+Wants=network-pre.target
+Conflicts=shutdown.target
+# Don't start when 'no-appfw` is in kernel command-line, to allow booting without it.
+ConditionKernelCommandLine=!no-appfw
 
 [Service]
-Type=simple
-PermissionsStartOnly=true
-ExecStartPre=/bin/mkdir -p /etc/opensnitchd/rules
-ExecStart=/usr/local/bin/opensnitchd -rules-path /etc/opensnitchd/rules
+Type=exec
+ConfigurationDirectory=%N/rules
+ConfigurationDirectoryMode=0700
+
+Environment='custom_cfg=%E/%N/rules'
+# Environment='opts=-debug'
+
+ExecCondition=%N -check-requirements
+ExecStart=%N -rules-path $custom_cfg $opts
```

Comment:
> @lainedfles 
> > The introduction of the _custom_config_ variable may be unnecessary unless use of `EnvironmentFile=` is planned. The structure of systemd makes it trivial to override `ConfigurationDirectory=` which, since v240, will expose the `$CONFIGURATION_DIRECTORY` environment variable automatically that we can re-use.
> 
> It can be removed when the functionality you mention is implemented, but until then lets keep it in the unit.


> +ExecStart=/usr/bin/%N -rules-path $CONFIGURATION_DIRECTORY $opts

Aha THAT's how you planned on using it?
Sure we could change it that way if you want... :woman_shrugging: 
- https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory= :+1: